### PR TITLE
Show options for video qualities.

### PR DESCRIPTION
**Pull request by meteor314**
Print options when user select anime quality.

### CODE REVIEW

File: ani-cli [hunk 1]
```diff
@@ -591,7 +591,7 @@ if [ -z "$select_first" ]; then
 			ep_choice_end=
 			;;
 		b)
-			prompt "Select quality"
+			prompt "Select quality. Options (best|worst|360|480|720|1080)"
```
Comment (ura43) [inline ani-cli:594]: For sake of consistency, I'd prefer `"Select quality. (opetions: best|worst|360|480|720|1080)"`

**Comment by ura43** [comment]
Also please edit your PR title to sth more descriptive.
"Add options reference to interactive quality selection" for example

**Comment by ura43** [comment]
Oh, almost forgot: please bump the version to 1.8.2

**Comment by meteor314** [comment]
when I want to choose the video quality, I don't have the options displayed, every  time I have to use the helpcommand to display the available options.
Here is an example :
Alt text 

Showing the options directly will avoid typing the help command alt text

**Review by ura43 — APPROVED** [review]

**Pull request merged**